src/features/game/components/bank/components/Withdraw.tsx

```diff
@@ -147,47 +167,75 @@ export const Withdraw: React.FC<Props> = ({ onClose }) => {
     );
   }
 
-  return (
-    <>
+  const mainMenu = () => {
```

Comment:
Try not to use function components. As much as possible try to create separate React components or just embed the components in each condition